src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -220,6 +239,22 @@ private async Task EnqueueProcessSnapshotWorkerAsync(Document document, Cancella
                     _reportChangeCancellationSource.Token);
             }
 
+            /// <summary>
+            /// TODO - Cleanup once experiment completed - https://github.com/dotnet/roslyn/projects/45#card-27261853
+            /// Only capture local classification telemetry for experiment when in liveshare and remote classifications are not active.
+            /// </summary>
+            private bool ShouldLogLocalTelemetry(string languageName)
+            {
+                var isLspContentType = _lspClientLanguages.Contains(languageName);
```

Comment:
Do we need to do any of the other stuff if this is already false? Just to skip all the extra code right away?